terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1 +1,11 @@
-# placeholder file
\ No newline at end of file
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
+
+# service configs
+log_level = "debug"
+cookie_domain = "cidev.aws.chdev.org"
```

Comment:
don't think cookie domain should be cidev

Reply:
looks like it will be along the lines of the stack name - https://github.com/companieshouse/overseas-entities-web/blob/main/terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars#L14
would check with toba

Reply:
Checked the other apps and this seems to what we use for staging and live. I have included @tobafolaranmi-ch on the PR.